### PR TITLE
[Merged by Bors] - feat(AlgebraicTopology): generalize universes for the standard simplex

### PR DESCRIPTION
**Pull request by joelriou**
The PR generalizes universes for `standardSimplex`, which is now a functor `SimplexCategory ⥤ SSet.{u}` for any universe `u`.

---

@jcommelin : we can wait you have finished showing that nerves of categories are quasi-categories before proceeding with this PR.

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/AlgebraicTopology/SimplicialSet.lean [hunk 1]
```diff
@@ -356,14 +410,24 @@ attribute [local simp] SSet.standardSimplex
...
     { left := SSet.standardSimplex.obj Δ
       right := terminal _
       hom := { app := fun Δ' => terminal.from _ } }
   map θ :=
     { left := SSet.standardSimplex.map θ
       right := terminal.from _ }
+  map_id n := by
+    ext1
+    · dsimp
+      erw [yoneda.map_id, uliftFunctor.map_id]
```
Comment (jcommelin) [inline Mathlib/AlgebraicTopology/SimplicialSet.lean:424]: Can we teach `aesop_cat` to take care of these proofs?

**Review by jcommelin — COMMENTED** [review]
Thanks for doing this! I don't think this has to wait on the nerve stuff.

File: Mathlib/AlgebraicTopology/SimplicialSet.lean (hunk 1 above)
Comment (joelriou) [reply Mathlib/AlgebraicTopology/SimplicialSet.lean]: Yes, I have just improved automation.

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**